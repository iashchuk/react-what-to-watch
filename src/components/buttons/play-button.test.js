import React from "react";
import renderer from "react-test-renderer";
import PlayButton from "./play-button.jsx";

describe(`PlayButton`, () => {
  it(`renders correctly`, () => {
    const dom = <PlayButton />;
    const component = renderer.create(dom);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

  it(`renders correctly with additional class`, () => {
    const dom = <PlayButton className="section__button" />;
    const component = renderer.create(dom);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
