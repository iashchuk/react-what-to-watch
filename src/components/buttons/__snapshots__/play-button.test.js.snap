// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlayButton renders correctly 1`] = `
<button
  className="btn btn--play"
  type="button"
>
  <svg
    height="19"
    viewBox="0 0 19 19"
    width="19"
  >
    <use
      xlinkHref="#play-s"
    />
  </svg>
  <span>
    Play
  </span>
</button>
`;

exports[`PlayButton renders correctly with additional class 1`] = `
<button
  className="btn btn--play section__button"
  type="button"
>
  <svg
    height="19"
    viewBox="0 0 19 19"
    width="19"
  >
    <use
      xlinkHref="#play-s"
    />
  </svg>
  <span>
    Play
  </span>
</button>
`;
